packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js

```diff
@@ -795,6 +796,32 @@ QUnit.module('items & dataSource', moduleConfig, () => {
 
             assert.strictEqual(this.getListItemsCount(), 0, 'was search');
         });
+
+        QUnit.testInActiveWindow('widget with fieldTemplate should not lose aria-required attribute after search and selection (T1230696)', function(assert) {
+            const $dropDownList = $('#dropDownList').dxDropDownList({
```

Comment:
Why you add test in `drop_down_list.tests.js` file, not `drop_down_editors.tests.js`?

Reply:
Draft of tests

```
QUnit.test('widget with fieldTemplate should not lose aria-required attribute after search and selection (T1230696)', function(assert) {
        const $selectBox = $('#dropDownEditorSecond').dxDropDownEditor({
            dataSource: ['one', 'two', 'three'],
            fieldTemplate: (data) => {
                return $('<div>').dxTextBox({ value: data });
            },
            valueChangeEvent: 'keyup',
        }).dxValidator({
            validationRules: [ { type: 'required' } ]
        });

        assert.strictEqual($selectBox.find(`.${TEXT_EDITOR_INPUT_CLASS}`).attr('aria-required'), 'true', 'initial render should have aria-required attribute set to true');

        keyboardMock($selectBox.find(`.${TEXT_EDITOR_INPUT_CLASS}`))
            .type('a');

        assert.strictEqual($selectBox.find(`.${TEXT_EDITOR_INPUT_CLASS}`).attr('aria-required'), 'true', 'aria-required should stay true after search and selection');

        keyboardMock($selectBox.find(`.${TEXT_EDITOR_INPUT_CLASS}`))
            .caret(1)
            .press('backspace');
       
        assert.strictEqual($selectBox.find(`.${TEXT_EDITOR_INPUT_CLASS}`).attr('aria-required'), 'true', 'aria-required should stay true after search and selection');
    });

    QUnit.test('widget with fieldTemplate should not lose aria-required attribute after search and selection (T1230696)', function(assert) {
        const markupRenderedStub = sinon.stub();

        const $selectBox = $('#dropDownEditorSecond').dxDropDownEditor({
            dataSource: ['one', 'two', 'three'],
            valueChangeEvent: 'keyup',
            fieldTemplate: (data) => {
                return $('<div>').dxTextBox({ value: data });
            },
            _onMarkupRendered: markupRenderedStub
        });

        assert.strictEqual(markupRenderedStub.callCount, 2, 'initial render should have aria-required attribute set to true');
        markupRenderedStub.reset();

        keyboardMock($selectBox.find(`.${TEXT_EDITOR_INPUT_CLASS}`))
            .type('a');

        assert.strictEqual(markupRenderedStub.callCount, 1, 'initial render should have aria-required attribute set to true');
        markupRenderedStub.reset();

        keyboardMock($selectBox.find(`.${TEXT_EDITOR_INPUT_CLASS}`))
            .caret(1)    
            .press('backspace');

        assert.strictEqual(markupRenderedStub.callCount,1, 'initial render should have aria-required attribute set to true');
    });
```